kiali-operator/values.yaml

```diff
@@ -81,3 +81,5 @@ cr:
     deployment:
       accessible_namespaces:
       - '**'
+
+  annotations: {}
```

Comment:
Please move this up to line 79 (place it above `spec`) so it doesn't confuse people thinking this is under "spec".
Also, please add a comment to `annotations: {}` to explain this to users - use it as a kind of documentation.

Something like:
```
  ...
  namespace: ""

  # Annotations to place in the Kiali CR metadata. 
  annotations: {}

  spec:
    ...
```

Reply:
Made the requested changes.